src/ophys_etl/modules/segmentation/modules/roi_comparison.py

```diff
@@ -0,0 +1,80 @@
+import pathlib
+import argschema
+
+from ophys_etl.modules.segmentation.qc_utils.roi_comparison_utils import (
+    create_roi_v_background_grid)
+
+
+class ROIComparisonSchema(argschema.ArgSchema):
+
+    background_paths = argschema.fields.List(
+            argschema.fields.InputFile,
+            cli_as_single_argument=True,
+            required=True,
+            description=("list of paths to pkl or png background images"))
+
+    background_names = argschema.fields.List(
+            argschema.fields.String,
+            cli_as_single_argument=True,
+            required=True,
+            description=("names of background images as they are to appear "
+                         "in plot"))
+
+    roi_paths = argschema.fields.List(
```

Comment:
perhaps a postload validation that the lengths of `roi_paths` and `roi_names` are the same.

Reply:
Done